embark.el

```diff
@@ -2851,7 +2922,9 @@ and leaves the point to the left of it."
   ("r" xref-find-references)
   ("b" where-is)
   ("e" pp-eval-expression)
-  ("a" apropos))
+  ("a" apropos)
+  ("n" search-forward)
```

Comment:
Nice! I like it, even though it doesn't search with symbol boundaries. I wonder if that'll be a problem.

Reply:
Yes, I figured that we prefer to use the builtins - also as a demonstration how Embark can power up built-in commands. In an older version I had the extra embark-symbol-next/previous commands.

Reply:
Right, and the prototype did add the symbol boundaries to the regexp. I like this, though, let's keep it and see if the symbol boundaries get requested.

Reply:
We can "maintain" the version with the boundaries in the wiki ;)

We will see - I am not yet sure how I am going to use all this repeat niceness. The n/p actions on symbols almost replace symbol-overlay. If we add more specialized next/previous actions (e.g. for urls or other targets), we may reconsider adding the proper symbol actions too. For which targets at point do we want such actions? @hmelman mentioned acting on the next target in #244.